docs/content/documentation/systems/Kernels/framework/BodyForce.md

```diff
@@ -1,7 +1,32 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # BodyForce
-!syntax description /Kernels/BodyForce
+
+## Description
+
+`BodyForce` implements a force term in momentum transport or structural
+mechanics or a source term in species/mass transport. Its weak form looks like
+$$(\psi_i, -f)$$ where $f$ is the magnitude of the force or source. The force is
```

Comment:
Can you update the $f$ to also discuss that this multiples value * pp * function.